soft4pes/model/grid/rl_grid.py

```diff
@@ -12,6 +12,19 @@ class RLGrid:
     The system input is the converter 3-phase switch position.
     The grid voltage is considered to be a disturbance.
 
+    Parameters
```

Comment:
three-phase switch position (instead of 3-phase...)

Reply:
Resolved in the next commit